codegenerator/cli/templates/dynamic/codegen/src/Types.res.hbs

```diff
@@ -219,12 +219,18 @@ external eventModWithoutArgTypeToInternal: module(Event) => module(InternalEvent
 
 {{#each codegen_contracts as | contract |}}
 module {{contract.name.capitalized}} = {
+  let contractName = "{{contract.name.capitalized}}"
+  let chains = [
+{{#each contract.chain_ids as | chain_id |}}
+  {{chain_id}},
+{{/each}}
+    ]->Belt.Array.map(chainId => ChainMap.Chain.makeUnsafe(~chainId))
```

Comment:
```suggestion
  ChainMap.Chain.makeUnsafe(~chainId={{chain_id}}),
{{/each}}
    ]
```

Reply:
Happy to take the suggestion but why?

This would expand into more boilerplate than before?

Reply:
I don't have a strong opinion about it